client/src/style/scss/ui.scss

```diff
@@ -452,3 +452,112 @@ $ui-margin-horizontal-large: $margin-v * 2;
     border-radius: 0.5rem;
     border: 0.1rem solid $brand-secondary;
 }
+
+// Form elements
+.ui-form-element {
```

Comment:
Overall I'd like to go the directon of removing global css, not adding to it.
Css can not be tree shaken, and global css makes it very easy to miss unused css styles.
Additionally, using the global name space can cause conflicts. Ideally, I think we should only have a small number of global utility classes, and have all other css co-located with their components.

Reply:
Thanks for the note. @ahmedhamidawan's first instinct was a different approach - moving the relevant styles into FormElement.vue. I led him down this wrong path.